tests/test_constants.py

```diff
@@ -0,0 +1,22 @@
+# The package atldld is a tool to download atlas data.
+#
+# Copyright (C) 2021 EPFL/Blue Brain Project
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+from atldld.constants import GLOBAL_CACHE_FOLDER
+
+
+def test_global_cache_folder():
+    assert GLOBAL_CACHE_FOLDER.exists()
```

Comment:
I am not a huge fan. I know that it is the current logic. However, a Python package should not test whether a user's filesystem contains a specific folder.

Maybe some patching here would be more appropriate

Reply:
Yeah, not a fan either 😆 like I said, maybe we can think about a better mechanism in a new issue. This PR was just to speed up the import  of `GLOBAL_CACHE_FOLDER` without changing the funtionality much.

We didn't use to create this folder, but I remember there was a bug in `atlalign` once that a function was trying to write to the cache folder that didn't exist, and I think we fixed it there by just creating a folder like this. I don't remember our code doing any checks on the existence of the cache folder, so I thought it was be safer that way. But we can drop it too if you guys think it's too much.

Like I said, I would suggest discussing better cache handling to a new issue/PR.

Reply:
Let's keep things the way they are and let's create a new issue:)

Reply:
(Conversation resolved - I do not have the button)